dts/arm/nordic/nrf52811.dtsi

```diff
@@ -50,6 +50,14 @@
 			label = "CLOCK";
 		};
 
+		radio: radio@40001000 {
+			compatible = "nordic,nrf-radio";
+			reg = <0x40001000 0x1000>;
+			interrupts = <1 NRF_DEFAULT_IRQ_PRIORITY>;
+			status = "okay";
+			dfe-supported;
```

Comment:
If DFE is added here, add also information about nRF52811DK in DF samples `README.rst`, please:
- https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/bluetooth/direction_finding_connectionless_rx/README.rst#requirements
- https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/bluetooth/direction_finding_connectionless_tx/README.rst#requirements

Reply:
> If DFE is added here, add also information about nRF52811DK in DF samples `README.rst`, please:

Can we instead say "nRF SoC with DFE support" or something like that? This way it is is more future-proof. We can tell users that they can check their devicetree for 'dfe-supported' in the radio node or check the product specification if they are unsur.

Reply:
>"nRF SoC with DFE support"
Good idea!